Makefile

```diff
@@ -1,7 +1,7 @@
 PROJECT = rabbitmq_cli
 
 BUILD_DEPS = rabbit_common
-TEST_DEPS = amqp_client
+TEST_DEPS = amqp_client rabbit
```

Comment:
Is this dependency really necessary? We try to avoid circular dependencies, e.g. see https://github.com/rabbitmq/rabbitmq-cli/commit/fc0ddac8a93c259176156ae9b932edfc66f19ddf. This may be OK for test dependencies but I don't see how the `rabbit` app is used in this PR?

Reply:
I built from a clean checkout (not in umbrella) and got an error that the rabbit module is missing due to this line:

https://github.com/rabbitmq/rabbitmq-cli/blob/master/lib/rabbitmq/cli/ctl/commands/wait_command.ex#L131

I'll open a PR to add that test dependency back.

Reply:
FYI here's the exact error:

```
** (UndefinedFunctionError) function :rabbit.await_startup/1 is undefined (module :rabbit is not available)
```

Reply:
This is a test dependency, although we do use rabbitmq-server code in several commands (they have `Validators.rabbit_is_loaded([], opts)` validator). The code is located using `RABBITMQ_HOME` environment variable.